### PR TITLE
 #536 Bugfix for empty lines appearing when switching tabs and some small UI changes

### PR DESCRIPTION
**Pull request by MelindaSW**
This gif illustrates the behavior of the bug. Switching tabs and scrolling makes the page blank and the position of the scroller is not correct.  
tab-switch-bug

And this is how it behaves after the fix.
tab-switch-bug-solution

The issue seems to be caused by the scrolling taking place before the component is fully rendered. After searching for similar issues I came across the option of adding of a timeOut on the scroll command. I tried it out and the behavior improved significantly. There is still a little bit of flicker as you can see in the gif which I can not seem to figure out how to disappear. 
But what do you think? Is this fix good enough for a merge and does it work on your machine?  

As a bonus, the settings icon is now moved to the upper right corner and the status bar color matches the rest of the app. 
image

### CODE REVIEW

File: src/js/view/components/LogViewerList.js [hunk 1]
```diff
@@ -82,7 +80,11 @@ const LogViewerList = props => {
         ref={listRef}
         items={props.lines}
         onRenderCell={_onRenderCell}
-        style={{ display: 'inline-block', minWidth: '100%' }}
+        style={{
+          display: 'inline-block',
+          minWidth: '100%'
+        }}
+        version={props.markedHighlight}
```
Comment (Mathriok) [inline src/js/view/components/LogViewerList.js:87]: Should this be here?

Comment (Atmanea) [reply src/js/view/components/LogViewerList.js:87]: Huh, that's weird, I wonder how it got there, pretty sure I stashed my changes before changing branch. I'll fix it now.

Comment (Atmanea) [reply src/js/view/components/LogViewerList.js]: Very good catch!!

Comment (MelindaSW) [reply src/js/view/components/LogViewerList.js]: Git works in mysterious ways

**Pull request merged**